tests/test_optimizers_pennylane.py

```diff
@@ -0,0 +1,46 @@
+import warnings
+import unittest
+
+import networkx as nx
+from openqaoa.workflows.optimizer import QAOA
+from openqaoa.devices import create_device
+from openqaoa.problems.problem import MinimumVertexCover
+from openqaoa.optimizers.training_vqa import CustomScipyGradientOptimizer
+
+
+#create a problem
+g = nx.circulant_graph(4, [1])
+problem = MinimumVertexCover(g, field =1.0, penalty=10)
+qubo_problem = problem.get_qubo_problem()
+
+
+class TestPennylaneOptimizers(unittest.TestCase):
```

Comment:
More exhaustive tests will be needed than the one we currently have, which only tests that the optimisation indeed ran. Some ideas would be:
- Repeat tests for another problem class
- Test using manual mode on top of of workflows
- Test that a step of the optimiser is indeed performed correctly with the input parameters and problem class (see e.g. `test_optimisers.py`, the `test_gradient_descent_step` test). Pennylane's implementation makes this easy, since there are the `step_and_cost` functions.